Scripts~/windows/.export-server-sdk.ps1

```diff
@@ -0,0 +1,49 @@
+param(
+	[Parameter(Mandatory)]
+	[Alias("ServerSdk", "SdkVersion", "Sdk")]
+	[ValidatePattern("\d+\.\d+\.\d+")]
+	[string] $ServerSdkVersion
+)
+
+$ROOT_DIR='.'
+$SERVER_SDK_FILENAME="GameLift-CSharp-ServerSDK-UnityPlugin-$ServerSdkVersion.zip";
+$DESTINATION_PATH="$ROOT_DIR\$SERVER_SDK_FILENAME"
+$TEMP_PATH="C:\Temp"
+$TEMP_ZIP_PATH = "$TEMP_PATH\GameLiftServerSDK.zip"
+$TEMP_EXTRACTED_PATH = "$TEMP_PATH\GameLiftServerSDK"
+$S3_SERVER_SDK_BUCKET="https://gamelift-server-sdk-release.s3.us-west-2.amazonaws.com/unity"
+$S3_SERVER_SDK_DOWNLOAD_LINK="$S3_SERVER_SDK_BUCKET/$SERVER_SDK_FILENAME"
```

Comment:
Technically, the bucket is just "gamelift-server-sdk-release". The link should be "https://$S3_SERVER_SDK_BUCKET.s3.us-west-2.amazonaws.com/unity/$SERVER_SDK_FILENAME".

Reply:
Renamed the variable